package.json

```diff
@@ -39,7 +39,7 @@
     "bootstrap": "yarn bootstrap:example && yarn bootstrap:integration-test",
     "bootstrap:example": "yarn && yarn example && yarn example pods && yarn brew",
     "bootstrap:integration-test": "yarn && yarn integration-test && yarn integration-test pods && yarn brew",
-    "brew": "[ \"$(uname)\" = \"Darwin\" ] && brew bundle install --no-lock",
+    "brew": "[ \"$(uname)\" != Darwin ] || brew bundle install --no-lock",
```

Comment:
Did you test this if we really don't need `"` around?

Reply:
:thinking: This is `sh`, you only need quotes if you have special characters. `Darwin` has none.